packages/react-error-overlay/src/components/overlay.js

```diff
@@ -50,11 +50,19 @@ function createOverlay(
   // Create header
   const header = document.createElement('div');
   applyStyles(header, headerStyle);
-  if (message.match(/^\w*:/)) {
-    header.appendChild(document.createTextNode(message));
-  } else {
-    header.appendChild(document.createTextNode(name + ': ' + message));
-  }
+
+  // Make message prettier
+  let finalMessage = message.match(/^\w*:/) ? name + ': ' + message : message;
```

Comment:
Fixed in https://github.com/facebookincubator/create-react-app/commit/2b596547c8e9f78122e2bed1d29ececc8b33f7a2